app/survey_config/survey_config.py

```diff
@@ -71,14 +71,7 @@ def get_footer_legal_links(  # pylint: disable=unused-argument, no-self-use
     ) -> Optional[list[dict]]:
         return None
 
-    def get_data_layer(self, tx_id: Optional[str] = None) -> list[dict]:
-        data_layer = [{"tx_id": tx_id}] if tx_id else []
-        if self.schema:
-            data_layer.append(
-                {
-                    key: self.schema.json[key]
-                    for key in ["form_type", "survey_id", "title"]
-                    if key in self.schema.json
-                }
-            )
-        return data_layer
+    def get_additional_data_layer_context(  # pylint: disable=no-self-use
```

Comment:
Why `self` in here when not used? Why create a method that always returns empty list (also wrong return type)?

Reply:
No I think this is correct, it will return the empty list because it is only used to return something when used in census but here we need to return nothing

Reply:
Yeah it is complicated type hint since we overwrite it in census config, we probably need to keep it as it is now.